pycodestyle.py

```diff
@@ -124,7 +124,7 @@
 STARTSWITH_DEF_REGEX = re.compile(r'^(async\s+def|def)')
 STARTSWITH_TOP_LEVEL_REGEX = re.compile(r'^(async\s+def\s+|def\s+|class\s+|@)')
 STARTSWITH_INDENT_STATEMENT_REGEX = re.compile(
-    r'^\s*({0})'.format('|'.join(s.replace(' ', '\s+') for s in (
+    r'^\s*({0})\s'.format('|'.join(s.replace(' ', '\s+') for s in (
```

Comment:
Isn't `\b` more appropriate here? For example, this is valid Python 3 (and should be flagged by this rule), but is unsupported by your regex:

```python
while+False: ...
```